src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCFs.java

```diff
@@ -0,0 +1,497 @@
+                int truthLen = truth.getAttributeAsInt(GATKSVVCFConstants.SVLEN, 0);
+                float diff = Math.abs(evalLen - truthLen);
+                return 1 - insLenDiffScoreCoeff * Math.max(diff/evalLen, diff/truthLen);
+            } else
+                return 0f;
+        } else if (evalType.equals(SimpleSVType.SupportedType.DEL.name())){
+
+            if (truthType.equals(SimpleSVType.SupportedType.DEL.name())) { // NOTE: one can use SVLEN, but remember the truth set might report positive SVLEN for DEL
+                SVInterval evalRange = new SVInterval(0, eval.getStart() - 1, eval.getEnd());
+                SVInterval truthRange = new SVInterval(0, truth.getStart() - 1, truth.getEnd());
+                float overlapLen = evalRange.overlapLen(truthRange);
+                return Math.min(overlapLen/evalRange.getLength(), overlapLen/truthRange.getLength());
+            } else if (truthType.equals(SimpleSVType.SupportedType.INS.name()) && eval.hasAttribute(GATKSVVCFConstants.INSERTED_SEQUENCE_LENGTH)) { // some call set might report replacement as deletion with insertion
+                final int truthInsLen = truth.getAttributeAsInt(GATKSVVCFConstants.SVLEN, 0);
+                final int evalInsLen = eval.getAttributeAsInt(GATKSVVCFConstants.INSERTED_SEQUENCE_LENGTH, 0);
+                float diff = Math.abs(evalInsLen - truthInsLen);
+                return 1 - insLenDiffScoreCoeff * Math.max(diff/evalInsLen, diff/truthInsLen);
+            } else {
+                return 0f;
+            }
+        } else if (evalType.equals(SimpleSVType.SupportedType.INV.name())) {
+
+            if (evalType.equalsIgnoreCase(truthType)) { // NOTE: one can use SVLEN, but remember the truth set might report SVLEN as positive or 0 for INV
+                SVInterval evalRange = new SVInterval(0, eval.getStart() - 1, eval.getEnd());
+                SVInterval truthRange = new SVInterval(0, truth.getStart() - 1, truth.getEnd());
+                float overlapLen = evalRange.overlapLen(truthRange);
+                return Math.min(overlapLen/evalRange.getLength(), overlapLen/truthRange.getLength());
+            } else {
+                return 0f;
+            }
+        } else { // TODO: 6/4/18 all BND types will be considered discordant, for now
+            return 0f;
+        }
+    }
+
+    @Override
+    public Object onTraversalSuccess() {
+        printSummaryToStdErr();
+        writeConcordanceToBedFile();
+        writeSkippedEvalBedFile();
+        return "SUCCESS";
+    }
+
+    // =================================================================================================================
+
+    // TODO: 8/8/18 a nice feature to add is stratification by some features, e.g. genomic context, length, etc., and possibly even allowing users to specify the stratification key and levels
+    private void printSummaryToStdErr() {
+
+        // header
+        System.err.print(String.format("%-25s:  ", "type"));
```

Comment:
Nope! Don't print to System.err. Use the logger instead!

Reply:
done